src/Components/ScheduleItem/index.tsx

```diff
@@ -9,6 +9,15 @@ export interface ScheduleItemProps {
   speaker: Speaker | null;
 }
 
+const UTC_FMT = new Intl.DateTimeFormat(
+  "en-GB-u-hc-h23",
```

Comment:
is this supported everywhere? might be worth adding a fallback, not everyone is on recent browsers :)

Reply:
It's reasonably widely supported, including IE 11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#Browser_compatibility

Reply:
ok cool! let's get this in 😊 thanks a lot!